docs/_posts/examples/SDK Build on Windows OS

```diff
@@ -0,0 +1,148 @@
+Windows OS - Windows 7 Professional (VM) 64bit
+
+NOTE: Windows uses backslash (\) in file path’s; while MAC uses forward slash (/)
+
+List of current install dependencies (find relevant download websites, and install these packages. Use NPM command line install where highlighted):
+
+•	Nodejs6.5
```

Comment:
Originally, I actually tried to build it with 4.x, but it did not work, that's when I upgraded to 6.5. I can try test it again with 4.x to get more specifics? Does the test suite get run on windows?